src/jdk.hotspot.agent/share/classes/sun/jvm/hotspot/debugger/windbg/WindbgDebuggerLocal.java

```diff
@@ -120,19 +120,8 @@ public void checkAlignment(long address, long alignment) {
     }
 
     if (useCache) {
-      // Cache portion of the remote process's address space.
-      // Fetching data over the socket connection to dbx is slow.
-      // Might be faster if we were using a binary protocol to talk to
-      // dbx, but would have to test. For now, this cache works best
-      // if it covers the entire heap of the remote process. FIXME: at
-      // least should make this tunable from the outside, i.e., via
-      // the UI. This is a cache of 4096 4K pages, or 16 MB. The page
-      // size must be adjusted to be the hardware's page size.
-      // (FIXME: should pick this up from the debugger.)
-      initCache(4096, 4096);
+        initCache(4096, parseCacheNumPagesProperty(1024 * 64));
```

Comment:
Not a review. Just a minor comment.
The indent does not match the above lines.
As I see, the indent in the updated files is not unified. There is even indent 3.